internal/gcsx/client_readers/range_reader.go

```diff
@@ -15,17 +15,38 @@
 package gcsx
 
 import (
+	"context"
+	"errors"
 	"fmt"
+	"io"
+	"math"
 
+	"github.com/googlecloudplatform/gcsfuse/v2/common"
+	"github.com/googlecloudplatform/gcsfuse/v2/internal/fs/gcsfuse_errors"
 	"github.com/googlecloudplatform/gcsfuse/v2/internal/gcsx"
 	"github.com/googlecloudplatform/gcsfuse/v2/internal/logger"
 	"github.com/googlecloudplatform/gcsfuse/v2/internal/storage/gcs"
+	"github.com/googlecloudplatform/gcsfuse/v2/internal/util"
+)
+
+const (
+	MiB = 1 << 20
+	// Max read size in bytes for random reads.
+	// If the average read size (between seeks) is below this number, reads will optimise for random access.
+	// We will skip forwards in a GCS response at most this many bytes.
+	// About 6 MiB of data is buffered anyway, so 8 MiB seems like a good round number.
```

Comment:
nit: Comment doesn't provide the full context. Either remove the last line, or be a bit more descriptive about the buffering. Here, we are talking about tcp buffer.

Note: I know, this comment was already there.

Reply:
Removed last line.